tests/scripts/components-configuration-crypto.sh

```diff
@@ -2353,11 +2353,22 @@ component_test_block_cipher_no_decrypt_aesce_armcc () {
     not grep aesce_decrypt_block ${BUILTIN_SRC_PATH}/aesce.o
 }
 
+component_test_ctr_drbg_aes_256_sha_512 () {
```

Comment:
Isn't this component using exactly the `full` config?

Reply:
Currently yes but not when we will switch to SHA-256 as the default hash for CTR_DRBG when both SHA-256 and SHA-512 are enabled.

Reply:
But then `component_test_ctr_drbg_aes_256_sha_256` will become identical to `full`. I don't understand why both components are present.

Reply:
Yes, it is just to ensure that we always test both SHA-256 and SHA-512 as CTR_DRBG hashes while we are doing changes in tf-psa-crypto. I will remove component_test_ctr_drbg_aes_256_sha_256 in a later PR when https://github.com/Mbed-TLS/TF-PSA-Crypto/pull/419 is merged. We will need another mbedtls PR anyway for some further clean-up related to the removal of MBEDTLS_ENTROPY_C in https://github.com/Mbed-TLS/TF-PSA-Crypto/pull/419.

Reply:
Ok, thanks for the explanation.